server.js

```diff
@@ -264,29 +266,10 @@ module.exports = function() {
       throw err;
     }
 
-    // We have these routes specifically for production where it's possible that
-    // a CDN index.html may refer to an outdated CSS/JS file that doesn't exist
-    if (process.env.NPM_CONFIG_PRODUCTION === 'true') {
```

Comment:
By removing the `/{params*}` now it requires that we enable this for both `development` and `production` for serving static files.